src/components/new-request/business-fetch.vue

```diff
@@ -6,28 +6,26 @@
       autocomplete="chrome-off"
       autofocus
       filled
-      @change="onItemSelected()"
-      @click:append="onItemSelected()"
-      :loading="state === States.SEARCHING"
-      :name="Math.random()"
       hide-details="auto"
       hint="Enter registration number of existing business"
       label="Fetch an existing business"
       persistent-hint
-      ref="searchField"
-      :rules="rules"
       v-model="searchField"
+      @click:append="onItemSelected()"
+      @keydown.tab.exact="onItemSelected()"
```

Comment:
This excludes Shift-Tab (so it searches when tabbing forward but not tabbing backward.